content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,158 @@
 =====================
-In-App Purchase (IAP)
+In-app purchase (IAP)
 =====================
 
-In-App Purchases (IAP) gives access to additional services through Odoo. For instance, it allows you
-to send SMS Text Messages or to send Invoices by post directly from your database.
+In-App Purchases (IAP) are optional services that enhance Odoo databases. For example, the **SMS**
+service sends text messages to contacts directly from the database, and the **Documents
+Digitization** service digitizes scanned or PDF vendor bills, expenses, and resumes with optical
+character recognition (OCR) and artificial intelligence (AI).
 
-.. _iap/buying_credits:
-
-Buying Credits
-==============
+IAP services do not need to be configured or set up, but each service requires its own prepaid
+credits. Each service has its own pricing and pack options listed on the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and Odoo users need to :ref:`buy more credits
+<iap/buying_credits>` when they run out, in order to keep using the service.
 
-Each IAP Service relies on prepaid credits to work and has its own pricing. To consult your current
-balance or to recharge your account, go to :menuselection:`Settings --> Odoo IAP --> View my
-Services`.
+.. tip::
+   Users with an Enterprise version of Odoo Online get free credits to test IAP features.
 
-.. image:: in_app_purchase/image1.png
+.. image:: in_app_purchase/iap.png
    :align: center
+   :alt: Visit IAP.Odoo.com to review what services are available.
 
-.. tip::
-   If you are on Odoo Online and have the Enterprise version, you benefit from free credits to test our
-   IAP features.
+.. _in_app_purchase/portal:
 
-IAP accounts
+IAP services
 ============
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+The `Odoo IAP website <https://iap.odoo.com/iap/all-in-app-services>`_ lists all IAP services
+currently available, which include the following:
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+- :guilabel:`AI generated call summary and transcript service`: call recording transcript service
+  for the *Asterisk Plus* application.
+- :guilabel:`Amazon Odoo Connector`: establishes seamless integration between an Amazon Seller
+  Central Account and Odoo, allowing users to effectively manage their products and orders.
```

Comment:
Hello, I don't think those service should be mentioned in our official documentation, they are provided by third parties.